staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/managedfieldsupdater_test.go

```diff
@@ -78,6 +292,26 @@ func (NoopManager) Update(liveObj, newObj runtime.Object, managed Managed, manag
 	return nil, nil, nil
 }
 
+func updateObject(t *testing.T, f *TestFieldManager, fieldManagerName string, object []byte) {
```

Comment:
Can you leave `t` out of these two functions (update and apply), return an error from these?